site/gatsby-site/server/local.ts

```diff
@@ -0,0 +1,57 @@
+import {
+    queryFields as quickAddsQueryFields,
+    mutationFields as quickAddsMutationFields,
+    permissions as quickAddsPermissions,
+} from './fields/quickadds';
+import { GraphQLObjectType, GraphQLSchema, GraphQLString } from 'graphql';
+import { shield, deny } from 'graphql-shield';
+import { applyMiddleware } from 'graphql-middleware';
+import { ObjectIdScalar } from './scalars';
+
+export const getSchema = () => {
+
+    const query = new GraphQLObjectType({
+        name: 'Query',
+        fields: {
+            ObjectId: {
+                type: ObjectIdScalar,
+                description: 'Custom scalar for MongoDB ObjectID',
+            },
+            ...quickAddsQueryFields,
+        }
+    });
+
+    const mutation = new GraphQLObjectType({
+        name: 'Mutation',
+        fields: {
+            ...quickAddsMutationFields,
+        }
+    });
+
+    const schema = new GraphQLSchema({
+        query,
+        mutation,
+    })
+
+
+    const permissions = shield(
+        {
+            Query: {
+                "*": deny,
+                ...quickAddsPermissions.Query,
+
+            },
+            Mutation: {
+                "*": deny,
+                ...quickAddsPermissions.Mutation,
+            },
+        },
+        {
+            allowExternalErrors: process.env.NODE_ENV !== 'production',
```

Comment:
By default, `graphql-shield` returns internal errors as `Not Authorized`, but we want to see Errors and stacktraces on development.